project-koku/koku
# [COST-3247] Properly handle manifest files from Azure

## Jira Ticket

[COST-3247](https://issues.redhat.com/browse/COST-3247)

## Description

Properly handle manifest files in Azure.

## Testing

1. Checkout Branch
```
git fetch upstream pull/4064/head:PR/4064
git switch PR/4064
```
2. Restart Koku
3. Add an Azure source using a partitioned export
4. Verify that data is properly processed.

## Notes

Once this is merged, the language in the docs stating _not_ to enable partitioning can be removed.
